packages/components/src/internal/components/domainproperties/samples/SampleTypePropertiesPanel.tsx

```diff
@@ -112,9 +117,21 @@ class SampleTypePropertiesPanelImpl extends React.PureComponent<
 
         this.state = {
             isValid: true,
+            containers: undefined,
         };
     }
 
+    componentDidMount() {
+        getValidPublishTargets()
+            .then(containers => {
+                this.setState({ containers });
+            })
+            .catch(response => {
+                console.error('Unable to load valid study targets for Auto-Link Data to Study input.');
```

Comment:
This was my best take on how to handle an unsuccessful request based on the other call that exists to `getValidPublishTargets()`

Reply:
Yup, that looks good.